keps/0008-node-heartbeat.md

```diff
@@ -0,0 +1,385 @@
+---
+kep-number: 8
+title: Efficient Node Heartbeat
+authors:
+  - "@wojtek-t"
+  - "with input from @bgrant0607, @dchen1107, @yujuhong, @lavalamp"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scalability
+  - sig-apimachinery
```

Comment:
I would add sig-scheduling, due to the image issue

Reply:
done